documentation/website/src/handbook/integration/serenity-2-migration-guide.md

```diff
@@ -0,0 +1,688 @@
+
+If you had a `Cast` implementation that set up the default abilities on your actors like this in your 1.x project:
+```javascript
+import { protractor } from "protractor";
+import { TakeNotes } from "serenity-js/lib/screenplay";
+import { Actor, BrowseTheWeb, Cast } from "serenity-js/lib/screenplay-protractor";
+
+export class Members implements Cast {
+    public actor(name: string): Actor {
+        const notepad = {};
+        return Actor.named(name)
+            .whoCan(BrowseTheWeb.using(protractor.browser))
+            .whoCan(TakeNotes.using(notepad));
+    }
+}
+
+```
+Then this will now need to be replaced by a version 2.x equivalent which now implements `DressingRoom`:
+
+```javascript
+import { Actor, DressingRoom } from "@serenity-js/core";
+import { BrowseTheWeb } from "@serenity-js/protractor";
+import { protractor } from "protractor";
+
+export class Actors implements DressingRoom {
+  prepare(actor: Actor): Actor {
+    return actor.whoCan(
+      BrowseTheWeb.using(protractor.browser),
+      TakeNotes.usingAnEmptyNotepad(),
+    );
+  }
+}
+```
+
+The above file will need to be placed in the `support` location specified for your test runner. See previous test runner sections where the specifics of each are explained.
+    
+Create a file called `configure_stage.ts` containing the following and place in the `support` location specified for your test runner:
+
+```javascript
+import { Serenity/JS } from "@serenity-js/core";
+import { Actors } from "../../screenplay/actors";
+
+beforeEach(function () {
+  this.stage = serenity.callToStageFor(new Actors());
+});
+```
+
+### Browser Synchronisation
+
+If your Serenity/JS project is testing a non-angular site you may have an `onPrepare` section in your protractor.config that contains the following:
+
+```javascript
+onPrepare: function () {
+    browser.ignoreSynchronization = true;
+},
+```
+If this is the case you will need to change this to the new syntax as follows:
+```javascript
+onPrepare: function () {
+    browser.waitForAngularEnabled(false);
+},
+```
+For more information on this topic, [see the docs](http://www.protractortest.org/#/api?view=ProtractorBrowser.prototype.waitForAngularEnabled)
+
+## Source file changes
+### Globally replacing imports
+If your project contains a large number of source files, quite a bit of time might need to be spent changing imports from the 1.x package structure to the 2.x equivalents. 
+If you tackle this on a per-file basis, the easiest approach might be to delete any invalid imports from the top of the source file, and allow your IDE to auto-detect and apply the correct imports. 
+However, this can get tedious very quickly as you'll find yourself repeating the same import process on many files. 
+
+The table below therefore itemises some common import changes that you are likely to need and you can use the "replace in path" function in your IDE to quickly make these changes across all of your sources files. 
+
+| from | to |
```

Comment:
This table doesn't render correctly on the website 🤔 
Not sure why...

Reply:
Looks like an issue with how Marked is invoked either by ESDoc or Metalsmith somewhere. I'll look into it.

Reply:
I think tables should work after the last commit. Can you try with the below syntax?

```
First Header | Second Header
------------ | -------------
Content Cell | Content Cell 
Content Cell | Content Cell 
```